adapters/feeder2core/feeder2core.go

```diff
@@ -31,6 +31,10 @@ func AdaptBlock(response *feeder.Block) (*core.Block, error) {
 		eventCount += uint64(len(response.Receipts[i].Events))
 	}
 
+	sigs := [][]*felt.Felt{}
+	if sig != nil {
```

Comment:
Do we need this check, or is it only for old tests?

Reply:
We need it, pending blocks dont have signatures.